### PR TITLE
Issue #20 | Added Prettier and ESLint

### PR DESCRIPTION
**Pull request by Niaz-Ul-Haque**
Added both the Prettier for better code and ESLint for updated Javascript. Both working fine, ran them once and fixed any bugs that occurred with it.

### CODE REVIEW

File: backend/.prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+   "printWidth": 100,
+   "singleQuote": true,
+   "trailingComma": "es5",
+   "tabWidth": 3,
```
Comment (ahmedsakr) [inline backend/.prettierrc.json]: had an interesting discussion with other collaborators about this but I think we'd like to keep this to 4 for now as a compromise on our preferences.

Comment (Niaz-Ul-Haque) [reply backend/.prettierrc.json]: ahh, will fix it to 4 tabs then asap. sorry and thank you for pointing it out!.

**Review by Niaz-Ul-Haque — COMMENTED** [review]
Requesting an updated PR very soon, review done. Thank you!

**Comment by ahmedsakr** [comment]
Just a heads up, a PR was merged yesterday and it seems your branch needs to be updated and resolve the merge conflicts!

**Comment by ahmedsakr** [comment]
also, do you mind starting a `.gitattributes` file in the root of the repository with the following content:

```
backend/package-lock.json binary
landing-page/package-lock.json binary
templates/package-lock.json binary
```

**Comment by Niaz-Ul-Haque** [comment]
As I was having difficulties merging, I issued a new PR with everything updated hopefully/ Please do a review and let me know/ Thank you very much! 
#22 (<- Link)

**Pull request closed without merging**